_includes/navbar.html

```diff
@@ -19,6 +19,7 @@
         <li><a href="/resources">Resources</a></li>
         <li><a href="/#orginizers">Organizers</a></li>
         <li><a href="/cfp">CFP</a></li>
+        <li><a href="{{site.baseurl}}/mission-statement">Mission Statement</a></li>
```

Comment:
My plan at the moment is to place this in the [body of the home page] (http://www.fullstack.la/#mission) and leave it off of the navbar.